### ISSUE TITLE
Add Auckland Transport Fare Zones

### ISSUE THREAD

**Issue by TLDuhamel** [consindo/waka#104]
Would be good to have the fare zones visible on the app.
https://at.govt.nz/media/1974237/simpler-fares-zone-map-web-june-2017.pdf

This could be done by adding a vector overlay, or by adding it to the tiles of the basemap - this could be possible with mapbox I think

**Comment by consindo** [comment]
Yeah, we can easily add an overlay. Thinking maybe only show it at higher zoom levels though - after all the stops disappear on the map, and perhaps just draw the edges on the zoomed in map.

**Comment by TLDuhamel** [comment]
Have seen that the Fare Zone area coordinates are included in the AT Journey Planner JS code at https://az609578.vo.msecnd.net/static/journeyplanner/v2/8d15fa8c06b1374542577e313ddc0c35ce79dec6/main.js

**Comment by consindo** [comment]
Annoyingly, the AT GTFS Feed doesn't include the correct Fare Zone Id - just comes up as merged_14. Metlink one seems to be accurate though.
